src/themes/greyVest/index.js

```diff
@@ -461,6 +462,11 @@ export let GVStyle = () => (
       .contexture-facet-cardinality {
         margin: 10px 0;
       }
+      .contexture-facet > label > div {
+        white-space: nowrap;
```

Comment:
Does this clamp it to one line? My understanding is that today, as long as there are spaces, it will wrap to new lines (which I think is the desired behavior)

Reply:
This does one line only and adds tooltip with the name and count on the row. 

![image](https://user-images.githubusercontent.com/910303/62593254-ddeff900-b88a-11e9-8699-1409a8a9c170.png)

VS 

![image](https://user-images.githubusercontent.com/910303/62593274-ee07d880-b88a-11e9-8acc-ced465cd97ed.png)

Can we confirm that the multi-line is what we want? It is just that line that needs to go and we should be good.